### DISCUSSION TITLE
Arena Points

### DISCUSSION THREAD

**Discussion by ThatLegend** [Prodigy-Hacking/ProdigyMathGameHacking#1177] [(C) Suggestions]
We need a hack for Arena points because when you use the win battle. so when you reload it does not save

**Comment by hostedposted** [comment]
I'm working on it. It's incredibly tedious.

**Reply by ThatLegend** [reply]
oh

**Reply by ThatLegend** [reply]
it is a good idea

**Comment by ThatLegend** [comment]
Any progress?

**Comment by ArcerionDev** [comment]
a solution is being worked on, it needs to be troubleshooted as it's not consistent yet.

**Reply by ThatLegend** [reply]
@ArcerionDev did u see Dev Stuff?????? (PART 2) i pinned you

**Comment by ThatLegend** [comment]
can you tell me when its done?
